asmcomp/cmm.mli

```diff
@@ -134,10 +134,10 @@ and expression =
   | Clet of Ident.t * expression * expression
   | Cassign of Ident.t * expression
   | Ctuple of expression list
-  | Cop of operation * expression list
+  | Cop of operation * expression list * Debuginfo.t
   | Csequence of expression * expression
   | Cifthenelse of expression * expression * expression
-  | Cswitch of expression * int array * expression array
+  | Cswitch of expression * int array * expression array * Debuginfo.t
```

Comment:
(Note that I'm not particulary against adding debuginfo to `Cassign`, it will probably help sometimes at the moment -- in the case where the `mov` happens to still be present in the output. But in principle I don't think it should be there, and in the long run I think that it should be removed -- or possibly `Cassign` itself should be removed)